### ISSUE TITLE
Log the timestamp with the loss in wandb

### ISSUE THREAD

**Issue by harshraj172** [yieldprotocol/mandrill#8]
During training , we send {'loss': 0.7057, 'learning_rate': 1.0507569011576136e-05, 'epoch': 2.84}. It will be useful to add timestamps which can help one ensure the run has not been hung.
